integration-tests/api/helpers/cart-seeder.js

```diff
@@ -50,6 +50,7 @@ module.exports = async (connection, data = {}) => {
   const r = manager.create(Region, {
     id: "test-region",
     name: "Test Region",
+    payment_providers: [{ id: "test-pay" }],
```

Comment:
Very nice 👌 did it raised a lot?

Reply:
A couple of tests were redundant without this addition, but should be good now 👍